conf/modules.config

```
@@ -89,7 +169,16 @@ process {
         ]
     }
 
+    withName: 'DROP.*FRASER_FITHYPERPARAMS' {
+        cpus = { 12 * task.attempt }
+        memory = { 70.GB * task.attempt }
+        time = { 2.h * task.attempt }
+    }
+
     withName: 'DROP.*FRASER_PSIVALUECALCULATION' {
+        cpus = { 30 * task.attempt }
```

from the report this process does not require that many cpus. Maybe 4?
```suggestion
        cpus = { 4 * task.attempt }
```